src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt

```diff
@@ -143,18 +144,48 @@ class ApplicationDataServiceOpenShift(
             )
         }
 
-        val deployDetails = createDeployDetails(dc)
+        fun getReplicationController(rcVersion: Long): ReplicationController? {
+            val rcName = "${dc.metadata.name}-$rcVersion"
+            return openshiftService.rc(namespace, rcName)
+        }
+
+        fun getRunningReplicationController(): ReplicationController? {
+            if (getReplicationController(dc.status.latestVersion)?.isRunning() == true) {
+                return getReplicationController(dc.status.latestVersion)
+            }
+            for (rcVersion in dc.status.latestVersion downTo 0) {
```

Comment:
Vil vi ikke her hente latestVersion på nytt enda en gang? Er downTo inclusivee der du starter eller exclusive?

Reply:
Godt poeng, det hadde jeg ikke tenkt på, men det er helt sant den vil bli gjort to ganger. Skal fikse.